src/controllers/commands/CommandController.cpp

```diff
@@ -938,6 +938,85 @@ void CommandController::initialize(Settings &, Paths &paths)
         return "";
     });
 
+    this->registerCommand("/mods", [](const auto & /*words*/, auto channel) {
+        auto formatError = [](HelixGetModeratorsError error, QString message) {
+            using Error = HelixGetModeratorsError;
+
+            QString errorMessage = QString("Failed to get moderators: ");
+
+            switch (error)
+            {
+                case Error::Forwarded: {
+                    errorMessage += message;
+                }
+                break;
+
+                case Error::UserMissingScope: {
+                    errorMessage += "Missing required scope. "
+                                    "Re-login with your "
+                                    "account and try again.";
+                }
+                break;
+
+                case Error::UserNotAuthorized: {
+                    errorMessage += "You must have moderator permissions to "
+                                    "use this command.";
+                }
```

Comment:
```suggestion
                case Error::UserNotAuthorized: {
                    errorMessage +=
                        "Due to Twitch restrictions, this command can only be "
                        "used by moderators. To see the list of mods you must "
                        "use the Twitch website.";
                }
```
Slightly reworded the error from the same case in `/vips`

Reply:
Corrected language